cloud/IoTHubListener/local.settings.json.temp

```diff
@@ -4,6 +4,7 @@
         "IoTHubConnectionString":"<intentionally blank>",
         "IoTHubEventHubEndpointConnectionString" : "<intentionally blank>",
         "EventHubEndpointConnectionString":"<intentionally blank",
+        "EventHubIngestConnectionString":"<intentionally blank",
```

Comment:
I like "ingest" here maybe push that ingest/egress verbiage across all our env variables? E.G. update line 6 above to cover "egress" connection string?